GameData/Astrogator/Localization/zh-cn.cfg

```diff
@@ -2,51 +2,56 @@ Localization
+		astrogator_landedError = <<1>> 已着陆。将其发射入轨, 以查看转移选项。 // <<1>> is landed. Launch to orbit to see transfers.
+		astrogator_highInclinationError = 轨道倾角为 <<1>>°; 当倾角超过 <<2>>° 时, 转移准确度过低。 // Inclination is <<1>>°, accuracy too low past <<2>>°
+		astrogator_noTransfersError = 没有可行的转移 // No transfers available
+		astrogator_launchSubtitle = 从 <<1>> 转移\n(发射所需Δv: 约 <<2>>) // Transfers from <<1>>\n(Launch ~<<2>>)
+		astrogator_transferColumnHeader = 转移 // Transfer
+		astrogator_timeColumnHeader = 距点火还有 // Time Till Burn
+		astrogator_deltaVColumnHeader = Δv // Δv
+		astrogator_durationColumnHeader = 点火时长 // Duration
+		astrogator_planetLabel = <<1>> // <<1>>
+		astrogator_maneuverButtonTooltip = 创建机动 // Create maneuver
+		astrogator_warpButtonTooltip = 时间加速至转移窗口 // Warp to window
+		astrogator_yearsValue = <<1>>年 // <<1>>y
+		astrogator_daysValue = <<1>>日 // <<1>>d
+		astrogator_hoursValue = <<1>>时 // <<1>>h
+		astrogator_minutesValue = <<1>>分 // <<1>>m
+		astrogator_secondsValue = <<1>>秒 // <<1>>s
+		astrogator_settingsButtonTooltip = 设置 // Settings
+		astrogator_backButtonTooltip = 返回 // Back
+		astrogator_closeButtonTooltip = 关闭 // Close
+		astrogator_columnHeaderTooltip = 排序 // Sort
+		astrogator_translationControlsNotification = 可使用平移控制 (H, N, I, J, K, L) 来调整机动节点 // Use translation controls to adjust nodes
+		astrogator_adjustManeuversMessage = 请调整机动, 以建立交会 // Adjust maneuvers to establish encounter
+		astrogator_manualLink = 点击此处查看在线手册 // Click for online manual
+		astrogator_manualLinkTooltip = README.md 文件中解释了每个设置项的含义 // The meaning of each setting is explained in the README.md file
+		astrogator_settingsSectionHeader = 设置: // Settings:
+		astrogator_planeChangeBurnsSetting = 生成改变轨道倾角的机动 // Generate plane change burns
+		astrogator_addChangeBurnsSetting = 将改变轨道倾角的机动计入Δv一栏 // Add plane change burns to Δv column
+		astrogator_autoDeleteNodesSetting = 自动删除玩家创建的机动节点 // Auto-delete user-created maneuver nodes
+		astrogator_asteroidsSetting = 计算已追踪小行星的转移窗口 // Calculate transfers to tracked asteroids
+		astrogator_maneuverCreationHeader = 机动节点创建: // Maneuver creation:
+		astrogator_autoTargetDestSetting = 自动将目的天体设为目标 // Automatically target destination
+		astrogator_autoFocusDestSetting = 自动聚焦视角至目的天体 // Automatically focus destination
+		astrogator_autoEditEjecSetting = 自动编辑出射机动节点 // Automatically edit ejection node
+		astrogator_autoEditPlaneChgSetting = 自动编辑改变轨道倾角机动节点 // Automatically edit plane change node
+		astrogator_autoSetSASSetting = 自动将SAS设置为机动节点模式 // Automatically set SAS to maneuver mode
+		astrogator_adjustNodesSetting = 当RCS关闭时, 可使用平移控制 (H, N, I, J, K, L) 来调整机动节点 // Adjust nodes with translation controls when RCS is off
+		astrogator_unitsHeader = 单位制: // Units:
+		astrogator_metricSetting = 国际单位制 (公制) // Système International d'Unités (Metric)
+		astrogator_imperialSetting = 美制单位 (英制) // United States Customary (Imperial)
+		astrogator_speedMetric = <<1>> m/s // <<1>> m/s
```

Comment:
I'd like to use a translation that means meters per second. Was "米/秒" not accurate?

Reply:
"米/秒" is accurate. 

In math and science textbooks in Chinese, unit symbols ("m/s"), rather than their translated names ("米/秒"), are almost always used. That's why I assumed the translated name was not neccesary for most Chinese players.

Hilariously, the Chinese translation of the stock game uses both ways. You can even see both at the same time on the same navball (which is pretty messed up!) Since both ways are "consistent" with the stock translation, "米/秒" will work fine if you prefer a translation.

![ms 米秒](https://github.com/user-attachments/assets/258730aa-ff62-4d28-b35d-0a8e1c1829cd)